language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/extensions/ballerina/syntaxhighlighter/BallerinaSyntaxHighlightService.java

```diff
@@ -23,9 +23,9 @@
 import java.util.concurrent.CompletableFuture;
 
 /**
- * An extension interface for Language server to fetch information about ballerina syntax hightlighter.
+ * An extension interface for Language server to fetch information about ballerina syntax highlighter.
  *
- * @since 1.0.2
+ * @since 1.1.0
  */
 @JsonSegment("ballerinaSyntaxHighlighter")
 public interface BallerinaSyntaxHighlightService {
```

Comment:
Do we need this? Since this is a notification as Diagnostics it seems it will be enough to add the ExtendedLanguageClient entry